test/cypress/integration/29-host.dashboard.test.js

```diff
@@ -38,10 +38,12 @@ describe('host dashboard', () => {
   });
 
   describe('Orders', () => {
-    it('mark pending order as paid', () => {
+    it('edit order and mark as paid', () => {
       cy.login({ redirect: '/brusselstogetherasbl/admin/orders' });
       cy.get('[data-cy="MARK_AS_PAID-button"]:first').click();
-      cy.getByDataCy('confirmation-modal-continue').click();
+      cy.get('[data-cy="amount-received"] input[name=amountReceived]').type('10.23');
```

Comment:
Great to have E2E tests for this :raised_hands:

Reply:
Thanks. 👍🏽